kytos/utils/napps.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, controller=None):
 
     def _load_kytos_configuration(self):
         """Request current configurations loaded by Kytos instance."""
-        uri = self._kytos_api + 'kytos/config/'
+        uri = self._kytos_api + 'api/kytos/core/config/'
```

Comment:
@cemsbr `self._config.get('kytos', 'api')` will be set to what? May be `/api` will be already there?

Reply:
@cemsbr ^